### PR TITLE
 Adding  the Solution of a problem 125 

### PR DESCRIPTION
**Pull request by parikhitritgithub**
## Description

 Adding  the Solution of a problem 125 

## Type of PR

Documentation update
Feature  update

## Checklist
 I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:

- [MDX Documentation](https://mdxjs.com/docs/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa-solutions/lc-solutions/0100-0199/0125-valid-palindrome.md [hunk 1]
```diff
@@ -27,17 +29,15 @@ Explanation: "amanaplanacanalpanama" is a palindrome.
...
 Input: "race a car"
 Output: false
 Explanation: "raceacar" is not a palindrome.
 ```
 
 ### Constraints
-
-- The input string may consist of uppercase and lowercase letters, digits, and punctuation symbols.
-- The empty string is considered a valid palindrome.
+- 1 <= s.length <= 2 * 105
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0100-0199/0125-valid-palindrome.md:39]: replace `- 1 <= s.length <= 2 * 105` to `- $1 \leq s.length \leq 2 \times 105$`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

File: dsa-solutions/lc-solutions/0100-0199/0125-valid-palindrome.md (hunk 1 above)
Comment (parikhitritgithub) [reply dsa-solutions/lc-solutions/0100-0199/0125-valid-palindrome.md]: Done

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**